src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/track-panel-variant/TrackPanelVariant.tsx

```diff
@@ -58,7 +58,9 @@ const TrackPanelVariant = (props: { focusVariant: FocusVariant }) => {
   };
 
   const variant = variantData.variant;
-  const mostSevereConsequence = <VariantConsequence variant={variant} />;
+  const mostSevereConsequence = (
+    <VariantConsequence variant={variant} withColour={true} />
```

Comment:
Mmm, I don't like how the default became without the colour 😞  The colour/consequence combination is likely to be more common (it's more common at the moment).

Reply:
Just changed that to `withoutColour` ✅

Reply:
Isn't it like `disabled` though?

Reply:
✅